client/src/main/kotlin/no/nav/su/se/bakover/client/skatteetaten/SkatteClient.kt

```diff
@@ -48,7 +46,7 @@ internal class SkatteClient(
     private val personOppslag: PersonOppslag,
     private val hentBrukerToken: () -> JwtToken.BrukerToken = { JwtToken.BrukerToken.fraMdc() },
     private val azureAd: AzureAd,
-    private val fnrOgListeAvSkattegrunnlagCache: Cache<Fnr, List<SamletSkattegrunnlagResponseMedYear>> = newCache(
+    private val fnrOgListeAvSkattegrunnlagCache: Cache<Fnr, List<SamletSkattegrunnlagForÅr>> = newCache(
```

Comment:
Kanskje vi skal bypasse denne cachen på en refresh? Vi har ikke like mye behov for cachen lenger siden vi lagrer og henter fra basen istedenfor å gå mot denne cachen hver gang.

Reply:
Dette kan fint være et eget kort og gjøres senere.